packages/otelbin/src/components/share/DownloadImageButton.tsx

```diff
@@ -39,19 +26,18 @@ export function DownloadImageButton() {
 
   const onClick = () => {
     const nodes = getNodes()
-    const nodesBounds = getRectOfNodes(nodes)
-    const { width, height } = calculateImageDimensions(nodes.length);
-    const transform = getTransformForBounds(nodesBounds, width, height, 0.5, 2)
+    const nodesBounds = getNodesBounds(nodes)
+    const {x, y, zoom} = getViewportForBounds(nodesBounds, nodesBounds.width, nodesBounds.height, 0.5, 2)
```

Comment:
**Question:** If we decrease the `min` and increase the `max`, can this result in a more useful image for cases in which a lot of pipelines are configured/very long pipelines are configured? Sample config: https://www.otelbin.io/s/8b947480-ecf0-47b3-895f-caccfde2660e

Reply:
Can you explain the `min` and `max` case? I didn't understand. I think this is already dynamically getting the required ViewPort based on boundaries. 

It can be useful to add borders and nothing goes wrong on image size, but at the same time if we increase or decrease the wrong values ​​we can distort the image view, right?

![collector-image (64)](https://github.com/dash0hq/otelbin/assets/37130241/44e4f7b5-fe7d-495f-ad8b-e998f72ce3ac)

Reply:
True! I was understanding that the last two parameters affect the min/max zoom level that the `getViewportForBounds` function may determine for its response. Consequently, I wasn't sure whether the `0.5` / `2` values are appropriate.

This being said, thank you for testing it. The result is looking nice! Will take a final look tomorrow (8pm here), but the state is looking very good! ❤️